client/state/initial-state.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import debugModule from 'debug';
+import localforage from 'localforage';
+import pick from 'lodash/object/pick';
+
+/**
+ * Internal dependencies
+ */
+import { createReduxStore } from 'state';
+import { pruneSites } from 'state/sites/reducer';
+import { pruneThemes } from 'lib/themes/reducers';
+
+/**
+ * Module variables
+ */
+const debug = debugModule( 'calypso:state' );
+
+const localforageConfig = {
+	name: 'calypso',
+	storeName: 'calypso_store',
+	description: 'Calypso Storage'
+};
+function pruneState( state ) {
```

Comment:
We may need to do this if we want to support immutable.js on some parts of the tree. serialize/unserialize works or I've used toObject/fromObject in the past.

What's here is currently a hack patch to get a proof of concept working. :)